docs-devsite/auth.auth.md

```diff
@@ -165,6 +166,33 @@ beforeAuthStateChanged(callback: (user: User | null) => void | Promise<void>, on
 
 [Unsubscribe](./util.md#unsubscribe)
 
+## Auth.initializeRecaptchaConfig()
+
+Changes the reCAPTCHA configuration on the `Auth` instance.
```

Comment:
s/Changes/Initializes

The below wording also needs edit since we are not accepting a recaptchaConfig anymore.